gossip-protocol-spec.md

```diff
@@ -152,29 +192,124 @@ struct Bloom {
 </details>
 
 ### PullResponse
-These are sent in response to a `PullRequest`.
+These are sent in response to a `PullRequest`. They contain filtered values from the node's `crds` which the origin node is missing. 
```

Comment:
careful with naming conventions. You can use origin however you want, but I would keep it consistent.